lib/daemon.js

```diff
@@ -28,7 +28,9 @@ export default function daemon() {
       process.stdout.write(`${JSON.stringify(result)}\n`);
     }).catch((error) => {
       // TODO: print entire stack trace here
-      process.stderr.write(error.toString());
+      process.stdout.write(JSON.stringify({
```

Comment:
I think this actually makes sense (printing to stdout). It makes the contract with the clients simpler, since they can expect a single line of JSON at all times. 

Ouch! Just realized there's no newline character at the end here. I'll add that if I hadn't already in a later commit.